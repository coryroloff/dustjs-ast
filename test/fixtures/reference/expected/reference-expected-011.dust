{one.two.three.four.five|js/}