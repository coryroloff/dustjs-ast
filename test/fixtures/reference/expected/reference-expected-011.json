{
	"body": [
		{
			"id": {
				"path": [
					{
						"name": "one",
						"loc": {
							"start": {
								"line": 1,
								"column": 2
							},
							"end": {
								"line": 1,
								"column": 5
							}
						}
					},
					{
						"name": "two",
						"loc": {
							"start": {
								"line": 1,
								"column": 6
							},
							"end": {
								"line": 1,
								"column": 9
							}
						}
					},
					{
						"name": "three",
						"loc": {
							"start": {
								"line": 1,
								"column": 10
							},
							"end": {
								"line": 1,
								"column": 15
							}
						}
					},
					{
						"name": "four",
						"loc": {
							"start": {
								"line": 1,
								"column": 16
							},
							"end": {
								"line": 1,
								"column": 20
							}
						}
					},
					{
						"name": "five",
						"loc": {
							"start": {
								"line": 1,
								"column": 21
							},
							"end": {
								"line": 1,
								"column": 25
							}
						}
					}
				],
				"loc": {
					"start": {
						"line": 1,
						"column": 2
					},
					"end": {
						"line": 1,
						"column": 25
					}
				}
			},
			"filters": [
				{
					"name": "js",
					"loc": {
						"start": {
							"line": 1,
							"column": 26
						},
						"end": {
							"line": 1,
							"column": 28
						}
					}
				}
			],
			"loc": {
				"start": {
					"line": 1,
					"column": 1
				},
				"end": {
					"line": 1,
					"column": 30
				}
			}
		}
	],
	"tokens": [
		{
			"type": "Punctuator",
			"value": "{",
			"loc": {
				"start": {
					"line": 1,
					"column": 1
				},
				"end": {
					"line": 1,
					"column": 2
				}
			}
		},
		{
			"type": "Key",
			"value": "one",
			"loc": {
				"start": {
					"line": 1,
					"column": 2
				},
				"end": {
					"line": 1,
					"column": 5
				}
			}
		},
		{
			"type": "Punctuator",
			"value": ".",
			"loc": {
				"start": {
					"line": 1,
					"column": 5
				},
				"end": {
					"line": 1,
					"column": 6
				}
			}
		},
		{
			"type": "Key",
			"value": "two",
			"loc": {
				"start": {
					"line": 1,
					"column": 6
				},
				"end": {
					"line": 1,
					"column": 9
				}
			}
		},
		{
			"type": "Punctuator",
			"value": ".",
			"loc": {
				"start": {
					"line": 1,
					"column": 9
				},
				"end": {
					"line": 1,
					"column": 10
				}
			}
		},
		{
			"type": "Key",
			"value": "three",
			"loc": {
				"start": {
					"line": 1,
					"column": 10
				},
				"end": {
					"line": 1,
					"column": 15
				}
			}
		},
		{
			"type": "Punctuator",
			"value": ".",
			"loc": {
				"start": {
					"line": 1,
					"column": 15
				},
				"end": {
					"line": 1,
					"column": 16
				}
			}
		},
		{
			"type": "Key",
			"value": "four",
			"loc": {
				"start": {
					"line": 1,
					"column": 16
				},
				"end": {
					"line": 1,
					"column": 20
				}
			}
		},
		{
			"type": "Punctuator",
			"value": ".",
			"loc": {
				"start": {
					"line": 1,
					"column": 20
				},
				"end": {
					"line": 1,
					"column": 21
				}
			}
		},
		{
			"type": "Key",
			"value": "five",
			"loc": {
				"start": {
					"line": 1,
					"column": 21
				},
				"end": {
					"line": 1,
					"column": 25
				}
			}
		},
		{
			"type": "Punctuator",
			"value": "|",
			"loc": {
				"start": {
					"line": 1,
					"column": 25
				},
				"end": {
					"line": 1,
					"column": 26
				}
			}
		},
		{
			"type": "Key",
			"value": "js",
			"loc": {
				"start": {
					"line": 1,
					"column": 26
				},
				"end": {
					"line": 1,
					"column": 28
				}
			}
		},
		{
			"type": "Punctuator",
			"value": "/",
			"loc": {
				"start": {
					"line": 1,
					"column": 28
				},
				"end": {
					"line": 1,
					"column": 29
				}
			}
		},
		{
			"type": "Punctuator",
			"value": "}",
			"loc": {
				"start": {
					"line": 1,
					"column": 29
				},
				"end": {
					"line": 1,
					"column": 30
				}
			}
		}
	],
	"loc": {
		"start": {
			"line": 1,
			"column": 1
		},
		"end": {
			"line": 1,
			"column": 30
		}
	}
}